src/lib/sfm/api_call_latency.py

```diff
@@ -0,0 +1,19 @@
+from lib.context import LoggingContext
+from collections import defaultdict
+
+
+class ApiCallLatency:
+    _value = defaultdict(list)
+
+    @staticmethod
+    def update(api_url, time):
+        if api_url in ApiCallLatency._value:
+            ApiCallLatency._value[api_url].append(time)
```

Comment:
out of this 4 lines, I think only 11 is needed if you use defaultdict. also in "print_statistics" when you reset _value it should be also defaultdict